VocaDb.ResXFileCodeGenerator/GroupResxFiles.cs

```diff
@@ -15,7 +15,10 @@ public static IEnumerable<GroupedAdditionalFile> Group(IReadOnlyList<AdditionalT
 			var baseName = Utilities.GetBaseName(path);
 			if (Path.GetFileNameWithoutExtension(path) == baseName)
 			{
-				lookup.Add(pathName + "\\" + baseName, file);
+				var key = pathName + "\\" + baseName;
+				//it should be impossible to exist already, but VS sometimes throws error about duplicate key added. Keep the original entry, not the new one
+				if (!lookup.ContainsKey(key))
```

Comment:
Hmm, would this be related to multithreading or something? Are you not sure why this happens?